### PR TITLE
univ.ml: Add assertion that all univs in the ucontext instance are globals

### PR DESCRIPTION
**Pull request by SkySkimmer**
There might also be vars in practice but let's see.

### CODE REVIEW

**Comment by SkySkimmer** [comment]
@coqbot ci minimize

**Comment by SkySkimmer** [comment]
Looks like metacoq also produces ucontext with var levels

File: kernel/univ.mli [hunk 1]
```diff
@@ -268,6 +268,8 @@ sig
   type t
 
   val make : Names.Name.t array -> Instance.t constrained -> t
+  (** The name array must be the same length as the instance, and the
+      instance must be all global names (no Set, no var) *)
```
Comment (ppedrot) [inline kernel/univ.mli:272]: This comment must be tweaked since we currently accept vars.

Comment (SkySkimmer) [reply kernel/univ.mli]: changed

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot run full ci

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**